ruleset.xml

```diff
@@ -1,28 +1,28 @@
 <?xml version="1.0"?>
 <ruleset name="Contributte">
-
-	<!-- Extending rulesets -->
-	<rule ref="./vendor/ninjify/coding-standard/ruleset.xml">
+	<!-- Contributte Coding Standard -->
+	<rule ref="./vendor/ninjify/coding-standard/contributte.xml">
+		<!-- Exclude some rules -->
 		<exclude name="Squiz.Commenting.FunctionComment.InvalidTypeHint"/>
-		<exclude name="SlevomatCodingStandard.ControlStructures.RequireTernaryOperator.TernaryOperatorNotUsed"/>
 	</rule>
 
 	<!-- Specific rules -->
 	<rule ref="SlevomatCodingStandard.Files.TypeNameMatchesFileName">
 		<properties>
-			<property name="rootNamespaces" type="array">
-				<element key="app/model" value="App\Model"/>
-				<element key="app/domain" value="App\Domain"/>
-				<element key="app/ui" value="App\UI"/>
-				<element key="app/modules" value="App\Modules"/>
-				<element key="tests/toolkit" value="Tests\Toolkit"/>
-				<element key="tests/cases" value="Tests\Cases"/>
-			</property>
+			<property name="rootNamespaces" type="array" value="
```

Comment:
Chceme pouzivat zapis s `<element>`. Proc jsi to upravil?

Reply:
OK. Někde je toto, jinde ono. Sjednotím u všech skeletonů